HHS/OPRE-OPS
# Ops 1036/task 1299

## What changed

Create a notification, which reflects the status of the deletion (success or fail)

## Issue

#1299 

## How to test

1. Goto the agreements list
2. Try and delete the first agreement, it should fail
3. Create a new agreement
4. Try and delete it, it should succeed
5. 💰 profit

## Screenshots

<img width="1188" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/b479914b-8309-4f86-8a09-b1da59a245b0">
